gobblin-config-management/src/main/java/gobblin/config/configstore/impl/ETLHdfsConfigStore.java

```diff
@@ -0,0 +1,165 @@
+package gobblin.config.configstore.impl;
+
+import gobblin.config.configstore.ConfigStoreWithImportedByRecursively;
+import gobblin.config.configstore.ConfigStoreWithResolution;
+import gobblin.config.configstore.ConfigStoreWithStableVersion;
+import gobblin.config.configstore.ImportMappings;
+import gobblin.config.configstore.VersionComparator;
+
+import java.net.URI;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+
+import com.typesafe.config.Config;
+import com.typesafe.config.ConfigFactory;
+
+
+/**
+ * ETLHdfsConfigStore is used for ETL configuration dataset management
+ * @author mitu
+ *
+ */
+public class ETLHdfsConfigStore extends HdfsConfigStoreWithOwnInclude implements ConfigStoreWithResolution,
+    ConfigStoreWithImportedByRecursively, ConfigStoreWithStableVersion {
```

Comment:
So far, this seems to use the default implementations for resolution and recursion, is there a point for it to extend the interfaces (it might give the wrong idea that it is doing things efficiently).